app/build.gradle

```diff
@@ -436,6 +436,18 @@ android {
         
     }
 
+    androidComponents {
+        onVariants(selector().withBuildType("release")) { variant ->
```

Comment:
Not sure we want to limit it to release builds. I think this is also useful for testing, we might want to have both mainland china and world versions.

Also we already have some `configureEach` loops for finetuning variants, couldn't we integrate this into any of those?

Reply:
Ping @felipeerias

Reply:
@svillar I have updated the PR so the changes apply to both release and debug builds.

Unfortunately the change in application Id needs to happen at a very specific time in the build process so I was not able to include it in our existing loops.